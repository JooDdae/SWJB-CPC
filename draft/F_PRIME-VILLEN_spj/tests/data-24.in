7159786761731759401962952271117376723935557385148777759543180891297976973099311332934671289402787073659239397434729641778978414603587952811171337245911297829405160898377547749371399758392036871846149931079437149580128519157145188075689677413326936863568957378861191106310931549352738637193400116999311392958499075387633741773361632346374991669467911355639811149832992391621773933723713121
