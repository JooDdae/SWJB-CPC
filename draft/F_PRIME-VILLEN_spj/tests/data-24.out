188
4 0 17 993 18 4 2 0 4 17 79 1189 3 1 1 4 0 0 0 4 2 4 19 52 2 3 3 3 4 534 93 4 4 4 17 3 14 0 1017 0 10 22 136 932 1152 2 2 10 2 91 0 1 24 557 1085 2 120 52 2 25 14 15 10 116 4 4 24 991 623 774 3 60 0 0 4 0 2 2 4 364 0 0 10 4 145 559 794 23 4 4 3 15 4 660 0 2 168 3 23 1141 885 1058 95 2 28 1169 35 761 414 1135 230 1097 749 19 4 13 2 2 57 2 883 749 3 4 2 4 1104 8 0 179 29 2 244 2 3 1 4 2 150 4 8 2 551 266 1152 2 158 768 155 710 825 13 4 4 2 2 803 1 2 90 4 95 263 633 0 0 2 3 740 141 35 23 10 1145 257 4 131 2 2 4 1 2 4 0 2 0 1 0 